rocky/rocky/views/tasks.py

```diff
@@ -80,7 +80,10 @@ def get_context_data(self, **kwargs):
         plugin_dict = {p.id: p.name for p in plugins}
 
         for task in task_list:
-            task.data.raw_data.boefje_meta.boefje.name = plugin_dict[task.data.raw_data.boefje_meta.boefje.id]
+            if task.data.raw_data.boefje_meta.boefje.id == "manual":
```

Comment:
You could extract `task.data.raw_data.boefje_meta.boefje.id` to a variable `boefje_id`. That would make it shorter and easier to read

Maybe for later: a `manual` property in the boefje meta that specifies wether this is a manual task would be nice, because we can then use it in both the backend as well as the templates and avoid the magic manual constant for the ID